### PR TITLE
Refine issue and PR templates

### LINKED ISSUE #2589: Issue templates are too "rigid"
### What is wrong?

It's quite rare that we can actually fit a simple issue within the existing templates.

### What should have happened?

Relax the 1 box fit all policy.  

### What machines are impacted?

All or N/A

### Steps to reproduce

Try to submit an issue that might be slightly out of the expected box it should fit in. 

### Additional information

Why is this field requiered?

### Do you have a proposed solution?

Templates are fine, but the one imposed here are way too rigid. This needs to be relaxed.

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
# Description
Refines the issue and PR templates to cover some shortcomings and pitfalls we have identified.

The fix file issue template is expanded to cover other data sets managed under "glopara".

Resolves #2589

# Type of change
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? NO

# How has this been tested?
N/A

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by aerorahul — DISMISSED** [review]

**Comment by DavidHuber-NOAA** [comment]
I rendered these changes in my fork:
https://github.com/DavidHuber-NOAA/global-workflow/issues/new/choose
https://github.com/NOAA-EMC/global-workflow/compare/develop...DavidHuber-NOAA:global-workflow:develop

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (DavidHuber-NOAA) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]: I think there is a bug in this template.  For some reason it is not rendering.

Comment (WalterKolczynski-NOAA) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: That's weird. It rendered originally and I didn't change it.

Comment (DavidHuber-NOAA) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: Sorry, commented on the wrong file.  It's the production update template.

File: .github/ISSUE_TEMPLATE/production_update.yml
Comment (DavidHuber-NOAA) [inline .github/ISSUE_TEMPLATE/production_update.yml:1]: Rather, this one doesn't render.

Comment (WalterKolczynski-NOAA) [reply .github/ISSUE_TEMPLATE/production_update.yml:1]: That makes more sense. Will fix.

Comment (WalterKolczynski-NOAA) [reply .github/ISSUE_TEMPLATE/production_update.yml:1]: Should be fixed now.

Comment (DavidHuber-NOAA) [reply .github/ISSUE_TEMPLATE/production_update.yml:1]: Yep, looks good now.

File: .github/ISSUE_TEMPLATE/static_data.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  - WalterKolczynski-NOAA
+
+body:
+  - type: dropdown
+    attributes:
+      label: Type of static data
+      description: |
+        - Fix: large static files needed to run global-workflow
+        - Initial conditions: ICs needed to run one of global-workflow's included tests
+        - Experimental data: Temporary datasets needed for development that will be replaced by other sources for operations (e.g. data that will be one DCOM)
```
Comment (DavidHuber-NOAA) [inline .github/ISSUE_TEMPLATE/static_data.yml]:
```suggestion
        - Experimental data: Temporary datasets needed for development that will be replaced by other sources for operations (e.g. data that will be in DCOM)
```

File: .github/ISSUE_TEMPLATE/static_data.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      required: false
+
+  - type: markdown
+    attributes:
+      value: |
+        # Tasks
+        - [ ] Discuss needs with global-workflow developer assigned to request
+        - [ ] Stage a CLEAN copy of data on Hera or WCOSS in the location provided above
+        - [ ] [global-workflow CM] Update data in "glopara"-managed space on supported platforms
+        - [ ] [Fix only] Update "Fix File Management" spreadsheet [https://docs.google.com/spreadsheets/d/1BeIvcz6TO3If4YCqkUK-oz_kGS9q2wTjwLS-BBemSEY/edit?usp=sharing]
```
Comment (DavidHuber-NOAA) [inline .github/ISSUE_TEMPLATE/static_data.yml]:
```suggestion
        - [ ] [Fix only] Update "Fix File Management" [spreadsheet](https://docs.google.com/spreadsheets/d/1BeIvcz6TO3If4YCqkUK-oz_kGS9q2wTjwLS-BBemSEY/edit?usp=sharing)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -23,27 +32,27 @@
   Refs #4321
   Refs NOAA-EMC/repo#5678
 -->
+<!-- For more on writing good commit messages, see https://cdeas.ms/git-commit/ -->
```
Comment (DavidHuber-NOAA) [inline .github/pull_request_template.md]: https://cdeas.ms/git-commit/ is a dead link.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good!

File: .github/ISSUE_TEMPLATE/static_data.yml [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        - "Fix: *NEW*"
+        - "ICs: C48/C48 mx500"
+        - "ICs: C96/C48"
+        - "ICs: C96 mx100"
+        - "ICs: C384/C192"
+        - "ICs: C384 mx025"
+        - "ICs: C768 mx025"
+        - "ICs: C1152 mx025"
+        - "ICs: *NEW RESOLUTION*"
+        - "Exp Data: GOCART emissions"
```
Comment (KateFriedman-NOAA) [inline .github/ISSUE_TEMPLATE/static_data.yml:57]:
```suggestion
        - "Exp Data: Global Dump Archive - new or experimental (RUNx.PDY)"
        - "Exp Data: GOCART emissions"
```

**Review by KateFriedman-NOAA — CHANGES_REQUESTED** [review]
Looks great, thanks for doing this, especially the static_data one!

File: .github/ISSUE_TEMPLATE/static_data.yml (hunk 4 above)
Comment (WalterKolczynski-NOAA) [reply .github/ISSUE_TEMPLATE/static_data.yml:57]: I originally had a dump requests as a separate category, but the info we need for that is different enough from the others it just made descriptions messy. I meant to make a new template for it, but forgot. Will do that in a follow-up PR.

Comment (KateFriedman-NOAA) [reply .github/ISSUE_TEMPLATE/static_data.yml:57]: Okie dokie, sounds good!

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks great, thanks @WalterKolczynski-NOAA !

**Pull request merged**